uber/site_sections/jobs.py

```diff
@@ -170,7 +170,7 @@ def staffers_by_job(self, session, id, message=''):
         return {
             'job':       job,
             'message':   message,
```

Comment:
I need to fix where it says ({'trusted': True}, it was just to make sure I didn't forget.  this code review is complete when all the new TODOs are gone.